pandas/__init__.py

```diff
@@ -80,6 +80,7 @@
     Float64Index,
     MultiIndex,
     IntervalIndex,
+    InvalidIndexError,
```

Comment:
not here, rather goes in pandas.errors

Reply:
Shall then I move the definition of `InvalidIndexError` (https://github.com/pandas-dev/pandas/blob/c71bfc36211b5e2d860a06d8fbef902b757bd6e4/pandas/core/indexes/base.py#L156) to `pandas.errors` module (https://github.com/pandas-dev/pandas/blob/master/pandas/errors/__init__.py)?

Reply:
yes, that's good